acceptance/bundle/templates/dbt-sql/output/my_dbt_sql/databricks.yml

```diff
@@ -3,7 +3,7 @@
 # See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
 bundle:
   name: my_dbt_sql
-  uuid: <UUID>
+  uuid: [UUID]
```

Comment:
I changed the convention here to `[` because `<` is escaped by the JSON marshaller (because it's an HTML character). Easier to change the convention than fix the sites where the encoding happens during assertions.

Reply:
https://github.com/databricks/cli/pull/2224